### ISSUE TITLE
LDN : February 2018

### ISSUE THREAD

**Issue by booyaa** [rust-community/talks#60]
If you can't do a talk this month what about next month: #60? 

- [ ] Date: **February 22nd @ 6:30PM**
- [ ] Theme: `None`
- Comms
  - [ ] Meetup page announced
  - [ ] Twitter
  - [ ] [Herald](http://herald.community.rs)
  - [ ] [URLO](https://users.rust-lang.org/c/community)
- [ ] Venue: **Skillsmatter**
- [ ] Sponsors
  - [ ] `Option`
- [ ] Speaker(s)
  - [ ] **@booyaa** - Announcements
  - [ ] Main Speaker (25m + 5m Q&A)
  - [ ] TBC Up to 3 x lightning talks (5-10m) 
- [ ] Post event write up on URLO.

**Announcements**
- [ ] `stuff.unwrap()`

# meetup.com text
```
Agenda:

• Welcome & networking
• News and Announcements
• Main Speaker: Name - Title of Talk

Lightning Speakers TBC Please PM me or @booyaa on twitter if interested

We're being hosted by Skillsmatter, so please remember to fill out their attendance link: __LINK__

Hashtag will be #LDN.RS

Speaker bios
```

**Comment by maciejkula** [comment]
I could do a 10-minute talk --- a blitz through [building an autodiff library in Rust](https://medium.com/@maciejkula/building-an-autodifferentiation-library-9ccf32c7a658).

**Comment by booyaa** [comment]
Yes please! I'll remove you off the February 22nd list unless you want do another new talk then? 😉

**Comment by booyaa** [comment]
Whoops ignore my comment, thought this was for January. Sorry!
